python/src/aiconfig/editor/client/src/LocalEditor.tsx

```diff
@@ -31,6 +26,31 @@ export default function Editor() {
     loadConfig();
   }, [loadConfig]);
 
+  const enableTelemetryIfAllowed = useCallback(async () => {
+    const res = await ufetch.post(ROUTE_TABLE.LOAD, {});
+
+    const enableTelemetry = res.allow_usage_data_sharing;
+
+    if (enableTelemetry) {
+      datadogLogs.init({
+        clientToken: "pub356987caf022337989e492681d1944a8",
+        env: process.env.NODE_ENV ?? "development",
+        service: "aiconfig-editor",
+        site: "us5.datadoghq.com",
+        forwardErrorsToLogs: true,
+        sessionSampleRate: 100,
+      });
+
+      const sessionID = generateUniqueSessionId();
+
+      datadogLogs.setGlobalContext({ session_id: sessionID });
+    }
+  }, []);
```

Comment:
setGlobalContext will add this as an attribute to every log sent

Reply:
also should clientToken be a constant somewhere? was unsure if that should be the case